src/sheerwater_benchmarking/utils/caching.py

```diff
@@ -114,10 +139,16 @@ def wrapper(*args, **kwargs):
 
                 # Store the result
                 if cache:
-                    if data_type == 'array':
+                    if ds is None:
+                        print(f"Autocaching null result for {null_path}.")
+                        with fs.open(null_path, 'wb') as f:
+                            f.write(b'')
+                            return None
+                    elif data_type == 'array':
                         write = False
                         if fs.exists(cache_path):
-                            inp = input(f'A cache already exists at {cache_path}. Are you sure you want to overwrite it? (y/n)')
+                            inp = input(f'A cache already exists at {
```

Comment:
Hm, thinking about whether we should have a human input in this? I

Reply:
I think this is pretty important or else people could be squashing each other's caches all the time. Especially without date awareness.

For instance without this if you run '1950' to '1960' it will just automatically overwrite the cache with nulls and nothing will stop it.